packages/shell-chrome/src/utils.js

```diff
@@ -55,51 +66,30 @@ export function flattenSingleAttribute(
     });
 
     if (Array.isArray(value)) {
-        margin = margin + 10;
-
-        for (var index in value) {
-
+        value.forEach((val, index) => {
+            const elementId = id ? id : attributeName;
             flattenSingleAttribute(
                 flattenedData,
                 index,
-                value[index],
-                typeof value,
-                margin,
-                (id ? id : attributeName) + "*" + index,
-                id ? id : attributeName
+                val,
+                typeof val,
+                margin + 10,
+                `${elementId}.${index}`,
+                elementId
             );
-        }
+        });
     } else if (value instanceof Object) {
-        margin = margin + 10;
-
-        for (var objectKey in value) {
+        Object.keys(value).forEach((objectKey) => {
```

Comment:
We can safely use `Object.entries()` here to get the index & value, since Chrome & Firefox both support it, removes the need for `value[objectKey]`.